libs/ui/src/reports/precinct_scanner_tally_report.tsx

```diff
@@ -89,6 +97,11 @@ export function PrecinctScannerTallyReport({
             );
           })}
         </TallyReportColumns>
+        {showQuickResults && (
```

Comment:
Wondering if we should just simplify this down to `signedQuickResultsReportingUrl &&` since we're already checking the polls transition status when determining whether or not to construct and pass this value